src/pybind/mgr/cephadm/tests/test_services.py

```diff
@@ -404,7 +404,7 @@ def test_nvmeof_config(self, _get_name, _run_cephadm, cephadm_module: CephadmOrc
 allowed_consecutive_spdk_ping_failures = 1
 spdk_ping_interval_in_seconds = 2.0
 ping_spdk_under_lock = False
-enable_monitor_client = False
+enable_monitor_client = True
```

Comment:
`src/pybind/mgr/cephadm/tests/test_services.py: fix nvmeof test`


There is no explanation for why. This commit message title is not acceptable: https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst#commit-title